src/formulas/tokenizer.ts

```diff
@@ -136,7 +136,7 @@ function tokenizeString(chars: string[]): Token | null {
   return null;
 }
 
-const separatorRegexp = /\w|\.|!|\$/;
+const separatorRegexp = /[\p{L}\p{N}_]|\.|!|\$/u;
```

Comment:
```suggestion
const separatorRegexp = /\p{L}|\p{N}|_|\.|!|\$/u;
```
to be consistent with the end part of the regex